pySDC/implementations/problem_classes/nonlinear_ODE_1.py

```diff
@@ -42,6 +49,9 @@ def __init__(self, u0=0.0, newton_maxiter=200, newton_tol=5e-11, stop_at_nan=Tru
             'u0', 'newton_maxiter', 'newton_tol', 'stop_at_nan', localVars=locals(), readOnly=True
         )
 
+        self.newton_itercount = 0

```

Comment:
Please use the work counter interface in the future so we can have a common interface across different problems. You can add an arbitrary number of them and record of them in a single hook.